tests/fsm/test_json_schema.py

```diff
@@ -552,10 +552,10 @@ class MockModel(BaseModel):
 
     # assert any ws pattern can be used
     if whitespace_pattern == "abc":
-        build_regex_from_object(MockModel, whitespace_pattern)
+        build_regex_from_schema(MockModel, whitespace_pattern)
```

Comment:
This seems to be broken independently of the current PR. `build_regex_from_schema` expects the schema and not the object. I fixed it.